cometd-java/cometd-java-server/cometd-java-server-http/cometd-java-server-http-jakarta/src/main/java/org/cometd/server/http/jakarta/JakartaCometDRequest.java

```diff
@@ -186,4 +194,23 @@ public String toString() {
             }
         }
     }
+
+    private record FailureListener(Consumer<Throwable> handler) implements AsyncListener {
```

Comment:
I'm nitpicking here, but a `record` doesn't look like the best choice for this purpose.